foxy/distribution.yaml

```diff
@@ -1652,6 +1652,22 @@ repositories:
       url: https://github.com/ros-drivers/phidgets_drivers.git
       version: dashing
     status: maintained
+  plotjuggler:
+    doc:
+      type: git
+      url: https://github.com/facontidavide/PlotJuggler.git
+      version: foxy
+    release:
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/facontidavide/plotjuggler-release.git
+      version: 2.8.4-1
+    source:
+      type: git
+      url: https://github.com/facontidavide/PlotJuggler.git
+      version: foxy
+    status: developed
+    status_description: developed
```

Comment:
This key should be removed. The `status` already describes the same. The description is usually only used when the status is EOL to provide context.

Reply:
@facontidavide friendly ping